apps/site/assets/ts/schedule/__tests__/LineDiagramTest.tsx

```diff
@@ -355,72 +344,3 @@ describe("LineDiagram for CR with branches going inward", () => {
     expect(moreStops.exists()).toBeTruthy();
   });
 });
-
-describe("LineDiagram live data", () => {
-  const animationFrameSpy = jest.spyOn(window, "requestAnimationFrame");
-  const documentHiddenSpy = jest.spyOn(document, "hidden", "get");
-
-  let wrapper: ShallowWrapper;
-
-  const mountComponent = () => {
-    wrapper = shallow(
-      <LineDiagram
-        lineDiagram={lineDiagram}
-        route={route as EnhancedRoute}
-        directionId={directionId}
-        routePatternsByDirection={
-          routePatternsByDirection as RoutePatternsByDirection
-        }
-        services={[]}
-        stops={{ 0: stops, 1: stops }}
-        today="2019-12-05"
-        scheduleNote={null}
-      />
-    );
-  };
-
-  it("polls for live data using an animation frame request", () => {
-    animationFrameSpy.mockImplementation(fn => {
-      fn(0);
-      return 1;
-    });
-    mountComponent();
-
-    expect(liveDataReload).toHaveBeenCalledTimes(1);
-  });
-
-  it("does not poll for live data when no animation frame occurs", () => {
-    animationFrameSpy.mockImplementation(fn => {
-      return 1;
-    });
-    mountComponent();
-
-    expect(liveDataReload).not.toHaveBeenCalled();
-  });
-
-  it("does not poll for live data when the page is not visible", () => {
-    animationFrameSpy.mockImplementation(fn => {
-      fn(0);
-      return 1;
-    });
-    documentHiddenSpy.mockReturnValue(true);
-    mountComponent();
-
-    expect(liveDataReload).not.toHaveBeenCalled();
-  });
-
-  it("does not poll for live data when another request is in flight", () => {
-    animationFrameSpy.mockImplementation(fn => {
-      fn(0);
-      return 1;
-    });
-    useFetch.mockImplementationOnce(() => ({
-      data: {},
-      isLoading: true,
-      reload: liveDataReload
-    }));
-    mountComponent();
-
-    expect(liveDataReload).not.toHaveBeenCalled();
-  });
-});
```

Comment:
These tests are simply deleted because they were testing the "glue code" we wrote to pull together `useInterval`, `useFetch`, and some other things. With SWR none of that glue is needed anymore, and the snapshot test combined with the mock data verifies that we are still displaying live data as we should (when it's present).